tests/cupy_tests/math_tests/test_misc.py

```diff
@@ -215,8 +215,8 @@ def test_nan_to_num_inf_nan(self):
 @testing.gpu
 @testing.parameterize(*testing.product({
     'mode': ['valid', 'same', 'full'],
-    'shape1': [(), (5,), (6,), (20,), (21,)],
-    'shape2': [(), (5,), (6,), (20,), (21,)],
+    'shape1': [(), (10,), (5,), (11,), (10,), (11,), (10,)],
+    'shape2': [(), (5,), (5,), (21,), (21,), (20,), (20,)],
```

Comment:
Why did you fix so?

Reply:
there are 3 (practically 6) cases that I wanted to test:
1 - first input has larger size
2 - second input has larger size ( the inverted case with the 4 combinations of even/odd lengths)
3 - same length

Reply:
`testing.prodcut` checks in all combinations of parameters (for example, checked in 75 cases in the parameter of the previous version). So you don't need to fix it.
On the contrary, your test parameters don't include `a1.size > a2.size and a2.size % 2 == 0` case.